internal/operators/cnv/cnv_operator.go

```diff
@@ -230,6 +229,12 @@ func (o *operator) GetPreflightRequirements(_ context.Context, cluster *common.C
 			},
 		},
 	}
+
+	if common.IsSingleNodeCluster(cluster) {
+		requirements.Requirements.Master.Quantitative.CPUCores += WorkerCPU
+		requirements.Requirements.Master.Quantitative.RAMMib += WorkerMemory
```

Comment:
Why?

Reply:
cause SNO is actually master and worker and in validation we check it though in UI we showed only master requirements without adding worker's